### PR TITLE
Fixed #30785 -- Doc'd that SESSION_COOKIE_DOMAIN supports subdomains.

### PR DESCRIPTION
**Pull request by GitRon**
#30785: Docs for `SESSION_COOKIE_DOMAIN` extended with hint about subdomain wildcard

### CODE REVIEW

File: docs/ref/settings.txt [hunk 1]
```diff
@@ -3032,6 +3032,10 @@ The domain to use for session cookies. Set this to a string such as
 ``"example.com"`` for cross-domain cookies, or use ``None`` for a standard
 domain cookie.
 
+Equivalent to :setting:`ALLOWED_HOSTS`, a value beginning with a period can be
+used as a subdomain wildcard: '.example.com' will match example.com,
+www.example.com, and any other subdomain of example.com.
```
Comment (felixxm) [inline docs/ref/settings.txt]: I don't think that we should mention `ALLOWED_HOSTS`, maybe a small rewording e.g.:
```
...
domain cookie. The setting also supports subdomains, so you could use
``".example.com"``, for example, to allow access from all subdomains of
``example.com``.
```

Comment (GitRon) [reply docs/ref/settings.txt]: Thanks, used your text. 👍

**Comment by felixxm** [comment]
@GitRon Thanks for this patch :+1: , however leading dot is not necessary anymore, sorry for confusing (see ticket [28741](https://code.djangoproject.com/ticket/28741) and afd375fc343baa46e61036087bc43b3d096bb0ca).

Closing per ticket.

**Comment by GitRon** [comment]
> 
> 
> @GitRon Thanks for this patch 👍 , however leading dot is not necessary anymore, sorry for confusing (see ticket [28741](https://code.djangoproject.com/ticket/28741) and [afd375f](https://github.com/django/django/commit/afd375fc343baa46e61036087bc43b3d096bb0ca)).
> 
> Closing per ticket.

Hi @felixxm I tried it and it is not working without the dot...

**Pull request closed without merging**